storybook/pages/CommunityNewCollectibleViewPage.qml

```diff
@@ -0,0 +1,52 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import AppLayouts.Chat.views.communities 1.0
+
+import Storybook 1.0
+import Models 1.0
+
+import utils 1.0
+
+SplitView {
+
+    Logs { id: logs }
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        Item {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            CommunityNewCollectibleView {
+                anchors.fill: parent
+                anchors.margins: 50
+                store: QtObject {
+                    property var layer1Networks: NetworksModel.layer1Networks
```

Comment:
is it better to have so many models for that? What about one model with some category role or sth?

Reply:
Will keep it as it is following the backend structure we have right now!